lib/glific/partners.ex

```diff
@@ -610,6 +611,22 @@ defmodule Glific.Partners do
     )
   end
 
+  @doc """
+  Determine if we need to show roles and permission for an organization
+  """
+  @spec get_roles_and_permission(map()) :: boolean
+  def get_roles_and_permission(organization),
+    do: FunWithFlags.enabled?(:roles_and_permission, for: %{organization_id: organization.id})
```

Comment:
I this this should retrun the organization.services.is_roles_and_permission right? 

Why we  are toggling it again here

Reply:
made requested changes